react/components/atoms/lightbox/lightbox.js

```diff
@@ -69,10 +131,152 @@ export class Lightbox extends PureComponent {
         this.setVisibility(true);
     };
 
+    onPanGesture = event => {
+        if (this.state.scale._value === 1 || !this.state.zoomed) return;
+
+        const dx = event.nativeEvent.translationX + this.translatedX;
+        const dy = event.nativeEvent.translationY + this.translatedY;
+        if (Math.abs(dx) < this.translatedXTreshold) {
+            this.state.translateX.setValue(dx);
+        }
+        if (Math.abs(dy) < this.translatedYTreshold) {
+            this.state.translateY.setValue(dy);
+        }
+    };
+
+    onPanGestureEnd = event => {
+        if (event.nativeEvent.state !== State.END) return;
+
+        this.translatedX = this.state.translateX._value;
+        this.translatedY = this.state.translateY._value;
+    };
+
+    onPinchGesture = event => {
+        const pinchScale = event.nativeEvent.scale;
+        const scale = this.baseScale._value * pinchScale;
+        if (scale < 1 || scale > 2) return;

```

Comment:
why these limits? For example, whats app seems to support a bigger scale.
Perhaps move the bigger value to a prop, so that it can be customizable